kolibri/plugins/coach/assets/src/views/common/resourceSelection/subPages/SelectFromTopicTree.vue

```diff
@@ -1,6 +1,6 @@
 <template>
 
-  <div>
+  <div v-if="topic">
```

Comment:
Were we getting to this point with topic=null? In what case does it happen? In the setup method we have other references to topic assuming its always defined

Reply:
Yes, on page refresh this can happen - I didn't observe any errors from the setup method though, only here. There's also the validation warning from the topic prop still extant, but that seemed valid.